src/rpc/blockchain.cpp

```diff
@@ -1137,12 +1128,15 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
             "  \"chainwork\": \"xxxx\"     (string) total amount of work in active chain, in hexadecimal\n"
             "  \"pruned\": xx,             (boolean) if the blocks are subject to pruning\n"
             "  \"pruneheight\": xxxxxx,    (numeric) lowest-height complete block stored\n"
-            "  \"softforks\": [            (array) status of softforks in progress\n"
+            "  \"softforks\": [            (array) status of buried softfork deployments\n"
             "     {\n"
             "        \"id\": \"xxxx\",        (string) name of softfork\n"
-            "        \"version\": xx,         (numeric) block version\n"
-            "        \"reject\": {            (object) progress toward rejecting pre-softfork blocks\n"
+            "        \"height\": \"xxxxxx\",  (numeric) height of the first block which the rules are enforced\n"
+            "        \"type\": \"xxxx\",      (string) original activation mechanism (\"ism\" or \"bip9\")\n"
+            "        \"version\": xx,         (numeric) block version (only for \"ism\" type)\n"
+            "        \"reject\": {            (object) progress toward rejecting pre-softfork blocks (only for \"ism\" type)\n"
```

Comment:
why not combine `['reject']['status']` with `['active']` below? Once these buried softforks are pinned, an active ISM softfork is equivalent to an active BIP9 softfork.

Reply:
No special reason. I just want to keep existing response untouched

Reply:
I honestly think `['reject']['status']` makes almost no sense in a post-ISM world. This PR is already a breaking API change (`getblockchaininfo` RPC no longer returns `bip9_softforks['csv']`) so why not use the opportunity to rationalize the way the information is presented to the user?

Reply:
ok. Then I prefer to just unify `softforks` and `bit_softforks`, so we don't need to break the API everytime we bury a softfork